src/Qdrant.Client/QdrantClient.cs

```diff
@@ -2361,6 +2361,8 @@ public async Task<UpdateResult> CreatePayloadIndexAsync(
 				PayloadSchemaType.Bool => FieldType.Bool,
 				PayloadSchemaType.Geo => FieldType.Geo,
 				PayloadSchemaType.Text => FieldType.Text,
+				PayloadSchemaType.Datetime => FieldType.Datetime,
+				PayloadSchemaType.Uuid => FieldType.Uuid,
```

Comment:
Not related to this PR, but a structural unit test would be good to assert that all `PayloadSchemaType` are handled, so that if a later version introduces a new one, the test will fail when the client is generated from the new version. Will open an issue to track.